server/test/api/accountType.js

```diff
@@ -0,0 +1,89 @@
+/*global describe, it, beforeEach, process*/
+
+var chai = require('chai');
+var chaiHttp = require('chai-http');
+var expect = chai.expect;
+chai.use(chaiHttp);
+
+var helpers = require('./helpers');
+helpers.configure(chai);
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
+
+var url = 'https://localhost:8080';
+var user = { username : 'superuser', password : 'superuser', project: 1};
+
+describe('The /account_types API endpoint', function () {
+  var agent = chai.request.agent(helpers.baseUrl);
+  var newAccountType = {
+    id : 4,
```

Comment:
[Actually, it is](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/models/schema.sql#L6).